libraries/Microsoft.Bot.Schema/DeliveryModes.cs

```diff
@@ -18,5 +18,6 @@ public static class DeliveryModes
     {
         public const string Normal = "normal";
         public const string Notification = "notification";
+        public const string BufferedReplies = "bufferedReplies";
```

Comment:
@johnataylor  Since this came from the swagger file, should we not update it with this option? https://github.com/microsoft/botframework-sdk/blob/master/specs/botframework-protocol/botframework-channel.json#L2293